addon/-private/properties/alias.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+import {
+  getProperty,
+  objectHasProperty
+} from '../helpers';
+
+/**
+ * Returns the value of some other property on the PageObject.
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continue: alias('continueButton.click')
+ * });
+ *
+ * // calls `page.continueButton.click`
+ * page.continue();
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continueText: alias('continueButton.text')
+ * });
+ *
+ * // checks value of `page.continueButton.text`
+ * assert.equal(page.continueText, 'Go ahead!');
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   clickContinueButton: clickable('button.continue'),
+ *   continue: alias('clickContinueButton')
+ * });
+ *
+ * // calls `clickContinueButton`
+ * page.continue();
+ *
+ * @public
+ *
+ * @param {string} pathToProp - dot-separated path to PageObject property
```

Comment:
Definitely an important point that `PageObject.alias` should be conceptually consistent with `Ember.computed.alias`. It seems like the `firstObject`/`lastObject` issue is mainly a difference between `PageObject.collection` and `Ember.Array` rather than a difference between `PageObject.alias` and `Ember.computed.alias`. Does that seem accurate?

If we have aliased methods return the alias context instead of the aliased method context, the helper name `alias` would be inaccurate, because in that case it's more than an alias...it's a transfer of ownership from one context to another. Hmmmmm. Maybe I need to rethink the name `alias`?